README.md

```diff
@@ -564,6 +564,60 @@ public class AWSSecretsManagerPluginSample {
 }
 ```
 
+## Read-Write Splitting Plugin
+
+The read-write splitting plugin adds functionality to switch between writer/reader instances via calls to the `JdbcConnection#setReadOnly` method. Upon calling `setReadOnly(true)`, the plugin will establish a connection to a random reader instance and direct subsequent queries to this instance. Future calls to `setReadOnly` will switch between the established writer and reader connections according to the boolean argument you supply to the `setReadOnly` method.
+
+### Loading the Read-Write Splitting Plugin
+
+The read-write splitting plugin is not loaded by default. To load the plugin, set the `connectionPluginFactories` connection parameter:
+
+```
+final Properties properties = new Properties();
+properties.setProperty("connectionPluginFactories", ReadWriteSplittingPluginFactory.class.getName());
+```
+
+If you would like to load the read-write splitting plugin alongside the failover and enhanced failure monitoring plugins, the read-write splitting plugin must be the first plugin in the connection chain, otherwise failover exceptions will not be properly processed by the plugin:
+
+```
+final Properties properties = new Properties();
+properties.setProperty(
+        "connectionPluginFactories",
+        String.format("%s,%s,%s",
+                ReadWriteSplittingPluginFactory.class.getName(),
+                FailoverConnectionPluginFactory.class.getName(),
+                NodeMonitoringConnectionPluginFactory.class.getName())
+```
+
+### Reader load-balancing
+
+The plugin can also load-balance queries among available reader instances by enabling the `loadBalanceReadOnlyTraffic` connection parameter. This parameter is disabled by default. To enable it, set the following connection parameter:
+
+```
+final Properties properties = new Properties();
+properties.setProperty("loadBalanceReadOnlyTraffic", "true");
+```
+
+Once this parameter is enabled, load-balancing will automatically be performed for reader instances when the connection has been set to read-only mode via `JdbcConnection#setReadOnly`. Load-balancing is performed by randomly selecting new reader instances at transaction boundaries.
+
+### Using the Read-Write Splitting Plugin against RDS/Aurora clusters
+
+When using the read-write splitting plugin against RDS or Aurora clusters, the plugin automatically acquires the cluster topology by querying the cluster. Because of this functionality, you do not have to supply multiple instance URLs in the connection string. Instead, users should only supply the URL for the initial instance they wish to connect to.
```

Comment:
How about:

Instead, supply just the URL for the initial instance to which you're connecting.

Reply:
Sounds good

Reply:
TY!